### PR TITLE
Made practice code optional

### PR DESCRIPTION
**Pull request by Vegemash**

### CODE REVIEW

File: protected/models/Contact.php [hunk 1]
```diff
@@ -70,8 +70,8 @@ public function rules()
         return array(
             array('nick_name', 'length', 'max' => 80),
             array('title, first_name, last_name, nick_name, primary_phone, qualifications, maiden_name, contact_label_id', 'safe'),
-            array('first_name, last_name', 'required', 'on' => array('manualAddPatient','referral','self_register','other_register','manage_gp','manage_practice')),
-            array('title', 'required', 'on' => 'self_register'),
+            array('first_name, last_name', 'required', 'on' => array('manualAddPatient','referral','self_register','other_register','manage_gp')),
+            array('title', 'required', 'on' => array('self_register', 'manage_practice')),
```
Comment (apigram) [inline protected/models/Contact.php]: I really don't think this needs to be mandatory.

Comment (Vegemash) [inline protected/models/Contact.php]: Something has to be mandatory on that form

Comment (apigram) [inline protected/models/Contact.php]: Wouldn't the address be mandatory though?

Comment (Vegemash) [inline protected/models/Contact.php]: Just having an address doesn't really name the practice

**Pull request closed without merging**